src/CommandAndStateMessageParser.cpp

```diff
@@ -379,10 +381,8 @@ base::samples::RigidBodyState CommandAndStateMessageParser::getCameraHeadTiltPos
     // here that the camera tilt position is equal to the camera tilt position command
     // (workaround)
     base::samples::RigidBodyState camera_head_tilt_position;
-    double camera_head_tilt_position_command_rad =
-        m_camera_head_tilt_position_command * M_PI / 180;
-    camera_head_tilt_position.orientation = Quaterniond(
-        AngleAxisd(camera_head_tilt_position_command_rad, Vector3d::UnitZ()));
+    camera_head_tilt_position.orientation =
+        Quaterniond(AngleAxisd(m_camera_head_tilt_position_command, Vector3d::UnitZ()));
```

Comment:
`tilt.elements[0].position => the position command in rad`
`m_camera_head_tilt_position_command => the position command in degrees`
`AngleAxisd => expects the position in rad`

Reply:
If AngleExisd expects a angle in rad, why m_camera_head should be in degree? Isn't it just a auxiliary variable?
the camera_head["tilt"]["position"] is in deegrees

Reply:
Actually, the error is that you are clamping `m_camera_head_tilt_position_command` using the boundaries as degrees instead of rads, that threw me off. `m_camera_head_tilt_position_command` should be rad indeed.

Reply:
Yes, i saw that, it was a mistake. Now it's ok